### ISSUE TITLE
Ethminer seemingly crashes on startup

### ISSUE THREAD

**Issue by knightshousegames** [ethereum-mining/ethminer#510]
I'm on Manjaro Linux, with a GTX 1070. After I get geth going, whenever I try to run...

`ethminer -G --opencl-platform 1` 
or 
`ethminer -U --opencl-platform 1` 
Or basically anything else other than
`ethminer -h` 

...Nothing happens. It just goes back to the terminal and gives no output anywhere. I've tried it in the Terminology terminal as well as the regular xfce terminal, and it gives me the same behavior. 

I wish I could give more, but it basically does nothing. I'm new to this, so I have no idea if I did something wrong, but the guides all make it seem like ethminer should be the easiest part.

**Comment by jasonwee** [comment]
enable verbosity should tell you more.

**Comment by knightshousegames** [comment]
Well, I got some help from the people over at the Manjaro forum and figured out an issue with the Manjaro Nvidia driver and Cuda version. Once that was resolved, it started reporting the following

`JSON-RPC problem. Probably couldn't connect.`

Where do we go from here?

**Comment by smurfy** [comment]
You need to specify the pool you want to use.

something like: ``-S eu1.ethermine.org:4444 -O <Your_Ethereum_Address>.<RigName>``

**Comment by chfast** [comment]
If you want to you your local geth, start it with `--rpc`  option.

**Comment by knightshousegames** [comment]
I'm already doing that, before I start the miner, in a separate tab in my terminal, I run

`geth --rpc --rpcaddr "localhost" --rpcport 8545 --fast`

Then I run

`ethminer -U -F http://127.0.0.1:8545 --opencl-platform 1`

I googled the error and thats what it said to do to fix it, and it didn't change anything

**Comment by jean-m-cyr** [comment]
Stupid question, but what does 'ping localhost' do?

Just making sure localhost resolves to 127.0.0.1

**Comment by knightshousegames** [comment]
It repeats something along the lines of this

`64 bytes from localhost (::1): icmp_seq=1 ttl=64 time=0.045 ms`

Over and over.

**Comment by chfast** [comment]
Just try `--rpcaddr 127.0.0.1`. It looks the localhost is ipv6 address.

**Comment by knightshousegames** [comment]
OK. so this is odd, but maybe normal?

I tried it once, it didn't work, then I was gonna get a screenshot so I started it up again, and geth gave me this error

`E0107 18:18:36.519047 p2p/nat/nat.go:110] Network port tcp:30303 could not be mapped: goupnp: SOAP request got HTTP 500 Internal Server Error`

Then it just started working. 

This time I started the miner before geth said "discover" and "sync". Was I just an idiot and starting the miner too late, or is this some wonderous computer miracle that I don't fully understand?
